report_xls/report_xls.py

```diff
@@ -29,13 +29,29 @@
 from openerp.report.report_sxw import report_sxw
 from openerp import pooler
 import logging
+
 _logger = logging.getLogger(__name__)
 
+xls_types_default = {
```

Comment:
Its purpose is to have a default in xls_types. You need to declare it before for that.